analysis/gosec.go

```diff
@@ -11,12 +11,12 @@ import (
 
 // GosecOutput is the struct that holds issues and stats found on a Gosec scan.
 type GosecOutput struct {
-	Issues []Issue
-	Stats  Stats
+	Issues []GosecIssue
```

Comment:
Wow, thanks a lot for this refactoring! Would you mind changing `Issues` to ` GosecIssues` and `Stats` into `GosecStats` as well? 

If so, would also change 
`for _, issue := range gosecOutput.Issues {`  into 
`for _, issue := range gosecOutput.GosecIssues {` ? 😄

Reply:
Sure can! Fixed variable names in 1dc1c00